lib/tiled.dart

```diff
@@ -1,35 +1,37 @@
 library tiled;
 
 import 'dart:convert';
-import 'dart:io';
-import 'dart:math';
 import 'dart:typed_data';
+import 'dart:ui';
 
 import 'package:archive/archive.dart';
 import 'package:xml/xml.dart';
 
+import 'src/parser.dart';
+
 part 'src/chunk.dart';
-part 'src/chunksize.dart';
-part 'src/editorsetting.dart';
-part 'src/enums.dart';
-part 'src/export.dart';
-part 'src/flips.dart';
-part 'src/frame.dart';
-part 'src/grid.dart';
+part 'src/common/enums.dart';
```

Comment:
Are there any pros over using part instead of export like we do in the other projects?

Reply:
the pro I can see is that you don't have to import all your internal files, so it was quite easy to work with.
but this is how it always worked on this package, I just kept unchanged